src/simplewallet/simplewallet.cpp

```diff
@@ -933,6 +934,10 @@ simple_wallet::simple_wallet()
   m_cmd_binder.set_handler("stop_mining",
                            boost::bind(&simple_wallet::stop_mining, this, _1),
                            tr("Stop mining in the daemon."));
+  m_cmd_binder.set_handler("set_node",
+                           boost::bind(&simple_wallet::set_node, this, _1),
+                           tr("set_node <nodeurl>[:<port>]"),
```

Comment:
Sorry again, but maybe `<nodeurl>` might be confusing since URL usually means something like `http://node.moneroworld.com`. Instead, `<host>` seems more appropriate here.

Reply:
Renamed ;)

Reply:
Should this actually be a URL? Shouldn't we have a way to distinguish current RPC vs ZMQ? What about using TLS, https?